feature-examples/python/api_key/create_api_key.py

```diff
@@ -0,0 +1,69 @@
+"""Example of using the Vectara REST API to create an API Key."""
+
+import logging
+from typing import Optional
+
+import requests
+
+
+def create_apikey(
+    customer_id: int,
+    corpus_id: int,
+    jwt_token: str,
+) -> tuple[Optional[str], bool]:
```

Comment:
remove `Optional` here because this never returns `None` in the first entry of the return tuple.